### DISCUSSION TITLE
How is this possible?

### DISCUSSION THREAD

**Discussion by dsmurrell** [miguelgrinberg/Flask-Migrate#499] [Q&A] [answered]
Your ask questions link lead me here: https://github.com/miguelgrinberg/Flask-SocketIO/discussions which is for another repo so I'll just leave this question here. How is this repo older than the alembic ones? Does alembic have an older repo?

![Screenshot 2022-12-05 at 09 42 42](https://user-images.githubusercontent.com/4035854/205605356-cff5b962-209b-41ee-a39c-1665750f2ee7.png)

**Comment by dsmurrell** [comment]
Nice, you fixed the link :)

**Comment by miguelgrinberg** [comment, marked as answer]
I have corrected the link to the discussions forum for this project. Thanks for letting me know.

The problem is that you are assuming star history matches the life of the repository, which is incorrect. You should look at commits if you want to learn about the history of the project, not stars. Alembic was hosted on bitbucket for a while in the early days I think.

**Comment by dsmurrell** [comment]
> Alembic was hosted on bitbucket for a while in the early days I think.

Cool, that explains it. Thanks.
